src/sknnr/datasets/_base.py

```diff
@@ -88,6 +88,53 @@ def _load_csv_data(
     return index, data, data_names
 
 
+def _load_dataset_from_filenames(
```

Comment:
What do you think about `_load_dataset_from_csvs`, just to be a little more specific? I don't have a strong opinion here, so whatever you prefer works.

Reply:
I tried to marry both our suggestions.  I went with `_load_dataset_from_csv_filenames`.  A bit long, but my brain wasn't reading `csvs` as the plural of `csv` - more like "cis-vis" :smile:.

Reply:
That's fair, looks good to me!